libmachine/provision/boot2docker.go

```diff
@@ -188,10 +188,6 @@ func (provisioner *Boot2DockerProvisioner) Provision(swarmOptions swarm.SwarmOpt
 		return err
 	}
 
-	if err := installDockerGeneric(provisioner); err != nil {
```

Comment:
I wasn't sure why this was even there.  B2D has Docker already and this seems like an unnecessary call to me.